lgc/test/scalarizationOfDescriptorLoadsTest1.ll

```diff
@@ -0,0 +1,113 @@
+; NOTE: Assertions have been autogenerated by tool/update_llpc_test_checks.py UTC_ARGS: --tool lgc
+; RUN: lgc -mcpu=gfx1010 -print-after=lgc-builder-replayer -o - %s 2>&1 | FileCheck --check-prefixes=CHECK %s
+; ModuleID = 'lgcPipeline'
```

Comment:
I think it would be really useful if you could add a comment that says what each test does, for example: "Make sure that there is a waterfall loop where..". Alternatively, use a more descriptive test name (file name).

Reply:
I added some comments at the top of the tests.